packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2148,10 +2211,6 @@ export class ContainerRuntime
 			case ContainerMessageType.Attach:
 				return this.dataStores.applyStashedAttachOp(opContents.contents);
 			case ContainerMessageType.IdAllocation:
-				assert(
```

Comment:
Can there be stashed ops for id compressor? If not, can we just assert that we shouldn't be here? If yes, then shouldn't they be processed in some way?

Reply:
There could be stashed ops, but due to nature of how ID compressor serializes its state (it serializes final state in a stash), it ignores them. Assert was valid before when we had two modes, now we have 3 modes. I'll add it back, but it will be different assert.